feature/session/src/main/res/layout/fragment_bottom_sheet_sessions.xml

```diff
@@ -49,12 +71,12 @@
             android:gravity="center_vertical"
             android:text="@{isFiltered ? @string/session_filter_button_filtered : @string/session_filter_button }"
             android:textColor="@color/gray2"
-            app:visibleGone="@{!isCollapsed}"
             app:chipBackgroundColor="@color/gray4"
             app:chipIcon="@drawable/ic_filter_list_black_24dp"
             app:chipIconTint="@color/gray3"
             app:layout_constraintBaseline_toBaselineOf="@id/sessions_bottom_sheet_title"
             app:layout_constraintEnd_toEndOf="parent"
+            app:visibleGone="@{!isCollapsed &amp;&amp; !shouldShowEmptyStateView}"
```

Comment:
It seems `sessions_bottom_sheet_show_filter_button` and `sessions_bottom_sheet_hide_filter_button` should be in the same view group like below because these views can be recognized as 2-state view.

```
        <FrameLayout
            android:id="@+id/sessions_bottom_sheet_toggle_filter_button_container"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            app:layout_constraintBottom_toBottomOf="@id/sessions_bottom_sheet_title" // baseline may not be used in this case
            app:layout_constraintEnd_toEndOf="parent"
            >
            <c.g.a.m.c.Chip
                android:id="@+id/sessions_bottom_sheet_show_filter_button"
                ....
                app:visibleGone="@{!isCollapsed}"
                />

            <c.g.a.m.c.Chip
                android:id="@+id/sessions_bottom_sheet_hide_filter_button"
                ...
                app:visibleGone="@{isCollapsed}"
                />
        </FrameLayout>
```

And also, don't forget to add `sessions_bottom_sheet_toggle_filter_button_container` to the constaint of nonEmptyStateViewGroup. These states are a little complicated 😂